File: solutions/cpp/darts/1/darts.cpp
#include "darts.h"
#include <cmath>
namespace darts {
double score(double x, double y) {
    if (pow(x,2) + pow(y,2) <= 100 && pow(x,2) + pow(y,2) > 25) {
        return 1;
    } else if (pow(x,2) + pow(y,2) <= 25 && pow(x,2) + pow(y,2) > 1) {
        return 5;
    } else if (pow(x,2) + pow(y,2) <= 1) {
        return 10;
    } else {
        return 0;
    }
}
} // namespace darts

File: solutions/python/darts/1/darts.py
def score(x, y):
    z = x ** 2 + y ** 2

    if z > 100:
        return 0
    if 25 < z <= 100:
        return 1
    if 1 < z <= 25:
        return 5
    if z <= 1:
        return 10
